kasa/smartprotocol.py

```diff
@@ -175,6 +176,7 @@ async def _execute_multiple_query(self, requests: dict, retry_count: int) -> dic
                 multi_result[method] = resp["result"]
             return multi_result
 
+        raise_on_error = end == 1
```

Comment:
This will not impact existing `smartprotocol` as it does not use `multipleRequest` when there is only one request. However the new camera protocol always sends `multipleRequest` for getter methods.

Reply:
If this is worth pointing out in the PR comments, perhaps it should be commented in the code, too?

Reply:
Done